projects/extension/src/content/ExtensionMessageRouter.ts

```diff
@@ -71,7 +71,11 @@ export class ExtensionMessageRouter {
 
     // tell the page when the port disconnects
     port.onDisconnect.addListener(() => {
-      sendMessage({ origin: "content-script", disconnect: true })
+      sendMessage({
+        origin: "content-script",
+        type: "error",
+        payload: "Disconnected!",
```

Comment:
I don't like this `payload`... Suggestions? @tomaka

Reply:
What happens is the extension has been installed or has decided by ban the application or something like that.
You can put "Lost communication with substrate-connect extension".

Reply:
yeah, I like that one better... I will update it, thanks!